pkgs/by-name/xd/xdcc-cli/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  pkgs,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "xdcc-cli";
+  version = "0.1.0";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "jreiml";
+    repo = "xdcc-cli";
+    rev = "v${finalAttrs.version}";
+    sha256 = "sha256-rPMerxQLB+/S3FtM4e5lCk0yAOUtWRhA8debwxbqmNI=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-qAQkKGyLEUR4Pf5KVjqBsdyFxsgD/docRWKrQsf7KVo=";
+
+  meta = with lib; {
+    description = "A CLI tool for downloading files using XDCC written in Rust";
+    homepage = "https://github.com/guilherme-n-l/xdcc-cli";
+    license = licenses.mit;
+    maintainers = with maintainers; [ jreiml ];
+    platforms = platforms.linux ++ platforms.darwin;
+    mainProgram = "xdcc-cli";
```

```suggestion
  meta = {
    description = "A CLI tool for downloading files using XDCC written in Rust";
    homepage = "https://github.com/guilherme-n-l/xdcc-cli";
    license = lib.licenses.mit;
    maintainers = with lib.maintainers; [  ];
    platforms = lib.platforms.unix;
    mainProgram = "xdcc-cli";
```
jreiml isn't a Nixpkgs maintainer, therefore she doesn't have a `lib.maintainers.jreiml` variable assigned. You must add yourself[^1] (most preferably) and/or another _willing_ person (optionally), who has a maintainer entry in Nixpkgs.

[^1]: might need to add an entry to `maintainers/maintainer-list.nix` in a separate commit (not a separate PR!) that precedes your edits